deepchecks/core/check.py

```diff
@@ -206,6 +212,79 @@ def _display_to_json(self) -> List[Tuple[str, str]]:
         matplotlib.use(old_backend)
         return displays
 
+    def to_wandb(self, wandb_init: bool = True, **kwargs: Any):
+        """Export check result to wandb.
+
+        Parameters
+        ----------
+        wandb_init : bool , default: True
+            if to initiate a new wandb run
+        kwargs: Keyword arguments to pass to wandb.init - relevent if wandb_init is True.
```

Comment:
Please add an Examples section that demonstrates an init with the important arguments of wandb

Reply:
it was resolved, I did add an example